res/controllers/Pioneer-DDJ-SX-scripts.js

```diff
@@ -381,6 +416,21 @@ PioneerDDJSX.init = function(id) {
             PioneerDDJSX.initDeck(index);
         }
     }
+
+    // init effects section:
+    PioneerDDJSX.effectUnit = [];
+    PioneerDDJSX.effectUnit[1] = new PioneerDDJSX.EffectUnit(1);
```

Comment:
Oh I'm sorry!!! I didn't saw that, damn!
I corrected the code as you said but now I get a new error:
"'this.getParameterIn' [undefined] is not a function."
What's this now?
My code:
```
    PioneerDDJSX.effectUnit = [];
    PioneerDDJSX.effectUnit[1] = new components.EffectUnit([1, 3]);
    PioneerDDJSX.effectUnit[2] = new components.EffectUnit([2, 4]);
    PioneerDDJSX.effectUnit[1].enableButtons[1].midi = [0x94, PioneerDDJSX.nonPadLeds.fx1on];
    PioneerDDJSX.effectUnit[1].enableButtons[2].midi = [0x94, PioneerDDJSX.nonPadLeds.fx2on];
    PioneerDDJSX.effectUnit[1].enableButtons[3].midi = [0x94, PioneerDDJSX.nonPadLeds.fx3on];
    PioneerDDJSX.effectUnit[1].effectFocusButton.midi = [0x94, PioneerDDJSX.nonPadLeds.fxTab];
    PioneerDDJSX.effectUnit[1].dryWetKnob.input = function(channel, control, value, status, group) {
        this.setParameterIn(this.getParameterIn() + PioneerDDJSX.getRotaryDelta(value) / 30);
    };
    PioneerDDJSX.effectUnit[1].init();
```